modules/gdscript/tests/scripts/runtime/features/stringify.gd

```diff
@@ -24,7 +24,8 @@ func test():
 	print(StringName("hello"))
 	print(NodePath("hello/world"))
 	var node := Node.new()
-	print(RID(node))
+	@warning_ignore("unsafe_call_argument")
+	print(RID(node)) # TODO: Why is the constructor (or implicit cast) not documented?
```

Comment:
What do you mean with this comment? https://docs.godotengine.org/en/stable/classes/class_rid.html#class-rid

Reply:
There is no `RID(Object)` constructor (or something like this).

Reply:
Ah right, we should document the explicit conversion.